index.html

```diff
@@ -9,12 +9,134 @@
+            <img class="graph__menu__icon" src="images/lightbeam_icon_help.png">
+            <img class="graph__menu__icon" src="images/lightbeam_icon_about.png">
+          </div>
+          <h1 id="filter">Recent Site</h1>
+          <h2 id="view">Graph View</h2>
+          <canvas id="canvas" width="300" height="300"></canvas>
+        </div>
+        <footer>
+          <div>
+            <h2>Toggle Controls</h2>
+            <div class="graph__controls">
+              <button class="footer-button active col-1">
+                <svg class="icon icon--button" version="1.1" xmlns="http://www.w3.org/2000/svg">
+                  <circle cx="8" cy="8" r="6"></circle>
+                </svg>
+                Visited Sites
+              </button>
+              <button class="footer-button active col-1">
+                <svg class="icon icon--button" version="1.1" xmlns="http://www.w3.org/2000/svg">
+                  <polygon points="0,14 7,2 14,14"></polygon>
+                </svg>
+                Third Party Sites
+              </button>
+              <button class="footer-button active col-1">
+                <svg class="icon icon--button" version="1.1" xmlns="http://www.w3.org/2000/svg">
+                  <line class="connection-line" x1="14" y1="2" x2="2" y2="14"></line>
+                </svg>
+                Connections
+              </button>
+              <button class="footer-button active col-2 row-1">
+                <svg class="icon icon--button" version="1.1" xmlns="http://www.w3.org/2000/svg">
+                  <circle cx="8" cy="8" r="6"></circle>
+                </svg>
+                <svg class="icon icon--button" version="1.1" xmlns="http://www.w3.org/2000/svg">
+                  <polygon points="0,14 7,2 14,14"></polygon>
+                </svg>
+                Watched Sites
+              </button>
+              <button class="footer-button active col-2 row-2">
+                <svg class="icon icon--button" version="1.1" xmlns="http://www.w3.org/2000/svg">
+                  <circle cx="8" cy="8" r="6"></circle>
+                </svg>
+                <svg class="icon icon--button" version="1.1" xmlns="http://www.w3.org/2000/svg">
+                  <polygon points="0,14 7,2 14,14"></polygon>
+                </svg>
+                Blocked Sites
+              </button>
+              <button class="footer-button active col-3 row-1">
+                <svg class="icon icon--button" version="1.1" xmlns="http://www.w3.org/2000/svg">
+                  <line class="cookies-line" x1="14" y1="2" x2="2" y2="14"></line>
+                </svg>
+                Cookies
+              </button>
+            </div>
+          </div>
+          <div>
+            <h2>Filter</h2>
+            <h2 class="hide">Hide</h2>
+            <div class="graph-filter">
+              <button class="footer-button button--filter active">Recent Site</button>
+              <button class="button--filter">Last 10 Sites</button>

```

Comment:
I have greatly simplified the button classes. Now there is just `active` and `double-icon`.